src/client/components/forms/publication.js

```diff
@@ -47,21 +55,41 @@ class PublicationForm extends React.Component {
 		this.handleSubmit = this.handleSubmit.bind(this);
 	}
 
+	/**
+	 * Changes the current tab to the one specified and checks
+	 * if the alias form is valid.
+	 *
+	 * @param {number} tab - Indicates the current tab
```

Comment:
I'd call this "the tab to be selected" - the current tab is ambiguous - it could be the one prior to selection.